v10.15/catalog/windows-machine-config-operator/catalog.json

```diff
@@ -257,86 +257,173 @@
+                "displayName": "Windows Machine Config Operator",
+                "installModes": [
+                    {
+                        "type": "OwnNamespace",
+                        "supported": true
+                    },
+                    {
+                        "type": "SingleNamespace",
+                        "supported": false
+                    },
+                    {
+                        "type": "MultiNamespace",
+                        "supported": false
+                    },
+                    {
+                        "type": "AllNamespaces",
+                        "supported": false
+                    }
+                ],
+                "keywords": [
+                    "windows"
+                ],
+                "maintainers": [
+                    {
+                        "name": "Red Hat, Windows Container Support for OpenShift",
+                        "email": "team-winc@redhat.com"
+                    }
+                ],
+                "maturity": "stable",
+                "minKubeVersion": "1.28.0",
+                "provider": {
+                    "name": "Red Hat"
+                }
             }
         }
     ],
     "relatedImages": [
         {
             "name": "",
-            "image": "registry.redhat.io/openshift4-wincw/windows-machine-config-operator-bundle@sha256:452a7634363c3f39aacab727c5cf7eef2c5bd95c39b5babb40a70b962f4f97c3"
+            "image": "quay.io/redhat-user-workloads/windows-machine-conf-tenant/windows-machine-config-operator/windows-machine-config-operator-bundle-release-4-15@sha256:8ca4e252fa170d92e4adc2603b220af7897cd6eb2ff9f808b05cc045e4ff1a9d"
```

Comment:
This should be registry.redhat.io/openshift4-wincw/windows-machine-config-operator-bundle@sha256:8ca4e252fa170d92e4adc2603b220af7897cd6eb2ff9f808b05cc045e4ff1a9